drools-wb-screens/drools-wb-scenario-simulation-editor/drools-wb-scenario-simulation-editor-kogito-runtime/src/main/java/org/drools/workbench/screens/scenariosimulation/webapp/client/dropdown/ScenarioSimulationKogitoRuntimeCreationAssetsDropdownProviderImpl.java

```diff
@@ -52,12 +52,20 @@ public void getItems(Consumer<List<KieAssetsDropdownItem>> assetListConsumer) {
         return response -> {
             List<KieAssetsDropdownItem> toAccept = response.stream()
                     .map(this::getKieAssetsDropdownItem)
+                    .filter(this::filterTargetDirectory)
```

Comment:
Is it possible to use a more advanced regexpr [here](https://github.com/kiegroup/drools-wb/blob/ecbde3e83c948a41fc983fad4cfb8238a3af3ae7/drools-wb-screens/drools-wb-scenario-simulation-editor/drools-wb-scenario-simulation-editor-kogito-runtime/src/main/java/org/drools/workbench/screens/scenariosimulation/webapp/client/dropdown/ScenarioSimulationKogitoRuntimeCreationAssetsDropdownProviderImpl.java#L37) instead of filter target after? I expect it should be enough something similar to `src/**/.*dmn`.

Reply:
@danielezonca I afraid that regexp it's not compatible with windows systems. https://issues.redhat.com/browse/KOGITO-1883. I think your suggestion can be applied in the moment we know how to deal with windows envs.

Reply:
@yesamer 
I had no time to look at the JIRA yet sorry...  you could try something like **/src/**/*.dmn to avoid having the target files in it.